app/services/task_status/make_link.rb

```diff
@@ -0,0 +1,41 @@
+module TaskStatus
```

Comment:
would it be helpful to add a spec for this class?

Reply:
i have amended the epic to ask that specs for these are added